### DISCUSSION TITLE
VS 2022 and CGAL compilation

### DISCUSSION THREAD

**Discussion by WindingWinter** [CGAL/cgal#6113] [Q&A] [answered]
Here [it says](https://doc.cgal.org/latest/Manual/windows.html) that

> CGAL 5.3 is supported for the following MS Visual C++ compilers: 14.0, 15.9, 16.0 (Visual Studio 2015, 2017, and 2019).

And now that [Visual Studio 2022](https://devblogs.microsoft.com/visualstudio/visual-studio-2022-now-available/) is out. Is VS 2022 supported also?

**Comment by lrineau** [comment, marked as answer]
Hi, VS 2022 is already supported by our development release, in the `master` branch of CGAL. It will be supported by CGAL-5.4, planned in about a month.

It is daily tested in our testsuite. You can see for example the results for today: [Test Results of CGAL-5.4-Ic-90](https://cgal.geometryfactory.com/CGAL/testsuite/results-5.4-Ic-90.shtml). The second column is `MSVC-2022-Preview-Release`: we have been testing MSVC 2022 since the first preview of it.
